lib/Doctrine/ORM/Cache/DefaultQueryCache.php

```diff
@@ -261,6 +261,14 @@ public function put(QueryCacheKey $key, ResultSetMapping $rsm, $result, array $h
             // @TODO - move to cache hydration components
             foreach ($rsm->relationMap as $alias => $name) {
                 $metadata = $this->em->getClassMetadata($rsm->aliasMap[$rsm->parentAliasMap[$alias]]);
+                $className = $metadata->getName();
```

Comment:
Shouldn't this change impact on https://github.com/doctrine/doctrine2/blob/master/tests/Doctrine/Tests/ORM/Cache/DefaultQueryCacheTest.php#L138?